terraform/a2ha-terraform/modules/s3/main.tf

```diff
@@ -43,15 +43,22 @@ resource "aws_iam_role_policy_attachment" "AdministratorAccess" {
 
 
 locals {
-  log_bucket = "${var.aws_s3_bucketName}-${var.random_id}"
+  log_bucket = "${var.aws_s3_bucketName}"
 }
 
-resource "aws_s3_bucket" "createS3bucket" {
-  bucket        = local.log_bucket
-  force_destroy = var.destroy_bucket
+# Creating s3 bucket using AWS-CLI (hab -> core/aws-cli)
+resource "null_resource" "createS3bucket" {
+  provisioner "local-exec" {
+    command = "chmod 700 ${path.module}/templates/createS3Bucket.sh; ${path.module}/templates/createS3Bucket.sh ${var.aws_s3_bucketName} ${var.aws_region}"
+  }
 }
 
-resource "aws_s3_bucket_acl" "elb_bucket_acl" {
-  bucket = local.log_bucket
-  acl    = "private"
-}
\ No newline at end of file
+# resource "aws_s3_bucket" "createS3bucket" {
```

Comment:
@ArvinthC3000  please remove this commented code out

Reply:
Done